x-pack/plugins/security_solution/common/api/endpoint/actions/list_route.ts

```diff
@@ -86,8 +86,12 @@ export const EndpointActionListRequestSchema = {
         }),
       ])
     ),
+    // action types
     types: schema.maybe(
-      schema.oneOf([schema.arrayOf(typesSchema, { minSize: 1, maxSize: 2 }), typesSchema])
+      schema.oneOf([
+        schema.arrayOf(actionTypesSchema, { minSize: 1, maxSize: 2 }),
```

Comment:
Wondering - can the `maxSize` be set to `RESPONSE_ACTION_TYPE.length` here so that we don't have to adjust the schema when new values are added?

Reply:
Updated here 1806c1e5ce814fba5c863b127875b90cfc7848ad. Also did the same for other params that used maxSize based on an array